src/app/app.component.ts

```diff
@@ -11,7 +11,8 @@ import {FormBuilder, FormGroup} from "@angular/forms";
                      [satDatepicker]="picker"
                      formControlName="date">
               <sat-datepicker #picker [rangeMode]="true"
-                              [closeAfterSelection]="false">
+                              [selectFirstDateOnClose]="true"
+                              [closeAfterSelection]="true">
               </sat-datepicker>
```

Comment:
Let's create a few datepickers to demonstrate all new features. (see current code)